### ISSUE TITLE
[Bug]:   Chinese Font is incorrect

### ISSUE THREAD

**Issue by svota** [hovancik/stretchly#1231]
### Version

- [X] I'm using version 1.12.0

### Known issues

- [X] I've checked Known issues

### Existing issues

- [X] I've checked Existing issues

### What operating system are you using?

Linux

### Operating System Version

Ubuntu 22.04.1 LTS

### Reproduction steps

1. Set language to simplified or traditional chinese
2. get a break message
3. notice that it is using Japanese characters for some words
...
**Reproduces how often:** [What percentage of the time does it reproduce?]

### Expected Behavior

Several characters have the same point in unicode for the Japanese and Chinese character, and the Noto Sans CJK KR and CJK JP favor the Japanese version while the CJK SC and CJK TC use the Chinese character. If my language is set to simplified or traditional Chinese, I would expect "睛" to render as 
![image](https://user-images.githubusercontent.com/424767/200903363-e1507d2f-208e-42bf-a4c2-a9190c0d70cd.png)

### Actual Behavior

Instead, it is rendered as ![image](https://user-images.githubusercontent.com/424767/200902831-79f37456-809d-40a6-a3da-fbed0c23bd1a.png) (this is true for a sizable subset of characters, and a known issue for Ubuntu: https://askubuntu.com/questions/901486/%E9%97%A8-looks-weird-on-my-system-default-font , https://forum.ubuntu.org.cn/viewtopic.php?t=478915 )

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by svota** [comment]
I see that I can update the font in app/css/common.css, but If I set it directly to CJK SC or CJK TC, this would just make the opposite problem appear in Japanese instead (and maybe korean? I'm not familiar with which set they prefer). Is the font being set per language somewhere else that I'm missing?

**Comment by hovancik** [comment]
Hi, I am loading multiple fonts in the `common.css`. I don't remember exactly, but I think browser (Stretchly uses electronJS and it uses Chromium) should detect the right font?

Here are the fonts: 

```
body {
  background: var(--main-bg-color);
  color: var(--main-color);
  font-family: 'Noto Sans Regular', 'Noto Sans Arabic Regular',
    'Noto Sans CJK KR Regular',
    'Noto Sans Devanagari Regular',
    'Noto Sans Gujarati Regular', 'Noto Sans Gurmukhi Regular',
    'Noto Sans Hebrew Regular',
    'Noto Sans JP Regular';
}
```

Seems to me we need to add  CJK SC and CJK TC font?

**Comment by svota** [comment]
Yeah, adding those fonts should do it, but I'm not 100% sure that electron will detect things, so we should test in both Japanese and Chinese after making the change.

**Comment by svota** [comment]
So here's a screencast showing the issue. I've set the language to Japanese (though the same will happen with Chinese). I think we need some sort of language toggle for these two fonts
[Screencast from 11-10-2022 11:42:29 AM.webm](https://user-images.githubusercontent.com/424767/201179992-55ca76b8-4d09-4da9-8384-c1ed0ab14a48.webm)

**Comment by svota** [comment]
Also, I made the change locally, but I'm not sure how to submit it. I looked through CONTRIBUTING.md, but it points to PULL_REQUEST_TEMPLATE, which doesn't exist?

**Comment by hovancik** [comment]
Hi, pull request template will appear once you make a PR on Github :) for example: https://github.com/hovancik/stretchly/pull/1154

**Comment by hovancik** [comment]
@svota were you able to figure it out? can I help?

**Comment by hovancik** [comment]
No reply and noone else complaining so closing.
